src/components/v5/common/ActionSidebar/partials/hooks.ts

```diff
@@ -81,3 +88,44 @@ export const useIsFieldDisabled = () => {
 
   return false;
 };
+
+export const useFinalizeSuccessCallback = (): FinalizeSuccessCallback => {
+  const { startPollingColonyData, stopPollingColonyData } = useColonyContext();
```

Comment:
Dropped you a DM also, but I think we can do this without polling and it should work just fine :ok_hand: